src/libs/recaptcha/webviewHTML.ts

```diff
@@ -10,6 +10,10 @@ export const reCaptchaWebviewHTML = `
             <title>ReCAPTCHA web view</title>
             <script>
                 let numberOfRetryRender = 0;
+                let userAgent = window.navigator.userAgent.toLowerCase();
+                let ios = /iphone|ipod|ipad/.test(userAgent);
+                let android = /android/.test(userAgent);
+                let isOnline = ios ? navigator.onLine : navigator.connection.type !== 'none';
```

Comment:
C'est pas toujours défini il me semble
```suggestion
                const isOnline = ios ? navigator.onLine : navigator.connection?.type !== 'none';
```